growth/too/models.py

```diff
@@ -842,8 +833,6 @@ class Observation(db.Model):
         db.DateTime,
         comment='Exposure timestamp')
 
-    field = db.relationship(Field)
```

Comment:
Why was this relationship removed?

Reply:
@lpsinger Why would we want it?

Reply:
If this change is not related to the TAP query, then it should be in a separate pull request.